public/app/plugins/panel/geomap/layers/data/geojsonDynamic.ts

```diff
@@ -88,11 +97,20 @@ export const dynamicGeoJSONLayer: MapLayerRegistryItem<DynamicGeoJSONMapperConfi
 
     const features = new ReplaySubject<FeatureLike[]>();
 
+    // Function to update feature properties for tooltip support
+    const updateFeatureProperties = (frame?: DataFrame) => {
```

Comment:
Was this wrapper added just to avoid calling `updateFeaturePropertiesForTooltip` with 4 arguments? Might be a bit confusing